src/%ZPM/PackageManager/Developer/File.cls

```diff
@@ -39,9 +39,11 @@ ClassMethod RemoveDirectoryTree(pRoot As %String) As %Status
 					$$$ThrowStatus(tSC)
 				}
 				Set pRoot = ##class(%Library.File).NormalizeDirectory(pRoot)
-				
-				Set tCmd = "ROBOCOPY "_tEmptyDir_" "_pRoot_" /MIR"
-				Set tSC = ##class(%ZPM.PackageManager.Developer.Utils).RunCommandViaZF(tCmd,.tLog,.tErr)
+				Set stdout=""
+				Set tCmd = $ListBuild("ROBOCOPY",tEmptyDir,pRoot,"/MIR")
+				Set tSC = ##class(%ZPM.PackageManager.Developer.Utils).RunCommand(, tCmd,.stdout,,.retCode)
+				if $$$ISERR(tSC),$Get(retCode)=1||($Get(retCode)=2) set tSC=$$$OK
```

Comment:
0       No errors occurred, and no copying was done.
            The source and destination directory trees are completely synchronized. 

    1       One or more files were copied successfully (that is, new files have arrived).

    2       Some Extra files or directories were detected. No files were copied
            Examine the output log for details. 

Why `$Get(retCode)=2`, as success ?

Reply:
Multiple testing in Windows 10 with the /MIR flag showed that all files are deleted.